components/form-builder/panel/QuestionInput.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useCallback, useEffect, useRef, useState } from "react";
+import styled from "styled-components";
+import { LocalizedElementProperties } from "../types";
+import debounce from "lodash.debounce";
+import { Input } from "./Input";
+
+const StyledInput = styled(Input)`
+  padding: 24px 10px 20px 10px;
+  border: none;
+  border-bottom: 1.5px solid #000000;
+  border-radius: 4px 4px 0 0;
+  font-weight: 700;
+  font-size: 20px;
+
+  &:focus {
+    border-color: #000000;
+    box-shadow: none;
+    background: #ebebeb;
+  }
+`;
+
+import { useTemplateStore } from "../store/useTemplateStore";
+import { useTranslation } from "next-i18next";
+
+export const QuestionInput = ({
+  index,
+  hasDescription,
+  initialValue,
+}: {
+  index: number;
+  hasDescription: string | undefined;
+  initialValue: string;
+}) => {
+  const { t } = useTranslation("form-builder");
+  const [value, setValue] = useState(initialValue);
+  const { localizeField, updateField, getFocusInput, setFocusInput } = useTemplateStore((s) => ({
+    localizeField: s.localizeField,
+    updateField: s.updateField,
+    focusInput: s.focusInput,
```

Comment:
Same as above: You can remove this line (we should never return this from the state anymore).